### PR TITLE
adding specific encoded keys for travis; using pyenv in travis

### PR DESCRIPTION
**Pull request by cmharlow**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,10 +26,10 @@ script:
 after_success:
 - true
 before_deploy:
-- pip install --upgrade pip
+- pyenv local 3.6.3
 - pip install pipenv
 - pipenv install
-- openssl aes-256-cbc -K $encrypted_ffb341899730_key -iv $encrypted_ffb341899730_iv -in .travis/deploy.zip.enc -out .travis/deploy.zip -d
+- openssl aes-256-cbc -K $encrypted_b2400a7690e2_key -iv $encrypted_b2400a7690e2_iv -in deploy.zip.enc -out deploy.zip -d
```
Comment (dkinzer) [inline .travis.yml]: Shouldn't this be referencing .travis/deploy.zip.enc and .travis/deploy.zip?

**Comment by cmharlow** [comment]
No because it writing out to deploy.zip

File: .travis.yml (hunk 1 above)
Comment (dkinzer) [reply .travis.yml]: I'm getting this error when I run on travis ```
deploy.zip.enc: No such file or directory
140265925867168:error:02001002:system library:fopen:No such file or directory:bss_file.c:398:fopen('deploy.zip.enc','r')
140265925867168:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:400:
unzip:  cannot find or open .travis/deploy.zip, .travis/deploy.zip.zip or .travis/deploy.zip.ZIP.
mv: cannot stat ‘.travis/.gritty_travis’: No such file or directory
mv: cannot stat ‘.travis/.conan_the_deployer’: No such file or directory
Agent pid 6260
chmod: cannot access ‘/home/travis/.ssh/.gritty_travis’: No such file or directory
chmod: cannot access ‘/home/travis/.ssh/.conan_the_deployer’: No such file or directory
/home/travis/.ssh/.gritty_travis: No such file or directory
/home/travis/.ssh/.conan_the_deployer: No such file or directory```

**Comment by cmharlow** [comment]
Where is that error in the Travis build?

**Comment by cmharlow** [comment]
It’s just not finding the file in this PR.

**Comment by dkinzer** [comment]
not a build.. I sshed in to the job to try it out before merging it.

**Comment by dkinzer** [comment]
It's still running into this issue ```bash .travis/deploy-qa.sh 
Cloning into 'tul_cob_playbook'...
Warning: Permanently added the RSA host key for IP address '192.30.253.113' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.
_even the ssh session... (even after running before_script successfully)._

**Comment by dkinzer** [comment]
It could be because there is a default .gitconfig set up with the following entry 
```
[user]
        name = Travis CI User
        email = travis@example.org
```

**Comment by dkinzer** [comment]
Adding this to .ssh/config works 
```
Host github.com
  IdentityFile ~/.ssh/.gritty_travis
```

But then it fails with
```
pyenv: version `3.7.2' is not installed (set by /home/travis/build/tulibraries/tul_cob_playbook/.python-version)
```

**Comment by cmharlow** [comment]
@dkinzer can you please merge the PR or otherwise so I can actually respond to errors in realtime?

**Comment by cmharlow** [comment]
it's pretty hard to debug via your comments as proxy.

**Comment by cmharlow** [comment]
otherwise, i'll just merge myself tomorrow AM so I can actually fix this.

**Pull request merged**